programs/survival/simply_harvest.sc

```diff
@@ -1,74 +1,77 @@
-///
-// Simply Harvest
-// by Gnottero & BisUmTo
-// (Carpet Mod 1.4.20)
+// 
+// by Gnottero
+// (Carpet Mod 1.4.169)
+// Allows the player to harvest crops by right-clicking (setting the age of the crop back to zero)
+// Compatible with minecraft 1.21 and with any crop + nether warts
 //
-// Allows the player to right-click on a crop to harvest it. The "Fortune" enchantment affects drops
-///
 
-__config() -> {'stay_loaded' -> true, 'scope' -> 'global'};
-
-global_seeds = {
-    'wheat' -> 'wheat_seeds',
-    'potatoes' -> 'potato',
-    'carrots' -> 'carrot',
-    'beetroots' -> 'beetroot_seeds',
-    'nether_wart' -> 'nether_wart'
+__config() -> {
+    'stay_loaded' -> true
```

Comment:
stay loaded is now implied, so no need to include non functional configs, right?

Reply:
> stay loaded is now implied, so no need to include non functional configs, right?

Oh, I didn't know it was default now. I'll fix modify it later

Reply:
I've updated the script removing the config block and also added another script, called simple_villagers, that allows the player to pickup villagers and place them back in the world